libexec/bats-core/bats-exec-test

```diff
@@ -52,6 +55,8 @@ load() {
     exit 1
   fi
 
+  # Dynamically loaded user files provided outside of Bats.
+  # shellcheck disable=SC1090
```

Comment:
https://github.com/koalaman/shellcheck/wiki/SC1090
 Ignoring as we cannot include files that do not exist yet.